From 5c34882e2010e8621d702060f0e1a34ded6b29f4 Mon Sep 17 00:00:00 2001
From: Josua Mayer <josua@solid-run.com>
Date: Mon, 4 Apr 2022 14:26:04 +0300
Subject: [PATCH 19/22] imx8mp: don't pre-define fdtfile environment variable

Signed-off-by: Josua Mayer <josua@solid-run.com>
---
 include/configs/imx8mp_solidrun.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/include/configs/imx8mp_solidrun.h b/include/configs/imx8mp_solidrun.h
index a4f37713fc..98e939a462 100644
--- a/include/configs/imx8mp_solidrun.h
+++ b/include/configs/imx8mp_solidrun.h
@@ -95,7 +95,6 @@
 	CONFIG_MFG_ENV_SETTINGS \
 	JAILHOUSE_ENV \
 	BOOTENV	\
-	"fdtfile=" CONFIG_DEFAULT_FDT_FILE "\0" \
 	"ramdisk_addr_r=0x50000000\0"	\
         "fdt_addr_r=0x43000000\0" \
         "fdt_addr=0x43000000\0" \
-- 
2.34.1

