From f6e14bebd58bb9e438c378d3475743490c093348 Mon Sep 17 00:00:00 2001
From: Yazan Shhady <yazan.shhady@solid-run.com>
Date: Thu, 5 Jan 2023 09:54:19 +0200
Subject: [PATCH] Disable PHY ADIN referance clock

---
 drivers/net/phy/adin.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/drivers/net/phy/adin.c b/drivers/net/phy/adin.c
index cf5a391c93e6..54720245805c 100644
--- a/drivers/net/phy/adin.c
+++ b/drivers/net/phy/adin.c
@@ -429,6 +429,9 @@ static int adin_config_init(struct phy_device *phydev)
 	if (rc < 0)
 		return rc;
 
+	/* Disable PHY referance clock (GE_REF_CLK_EN=0) */
+	phy_modify_mmd(phydev, 30, 0xff1f, 0b000111111, 0);
+
 	phydev_dbg(phydev, "PHY is using mode '%s'\n",
 		   phy_modes(phydev->interface));
 
-- 
2.39.2

